apps/sensenet/cypress/integration/ui/drawerActiveStates.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Drawer menu icons', () => {
+  beforeEach(() => {
+    cy.login()
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+  })
+
+  it('clicking on the Search icon on the drawer should open the Search page', () => {
+    cy.get('[aria-label="Search"]').as('searchIcon')
```

Comment:
please select by a custom attribute (e.g. data-test) instead of aria-label. I know, I know there's no data-test attribute on the item, so please add in /src/components/drawer/PermanentDrawer.tsx, its value could be the same as the value of the aria-label 
![image](https://user-images.githubusercontent.com/6968860/95862308-057e0380-0d63-11eb-8f67-f89e7ca8f668.png)